tests/framework/PostProcessors/SafestPoint/test_safest_point_cdf.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<Simulation verbosity="debug">
+<Simulation verbosity="silent">
```

Comment:
For this and other verbosity switches in the pull, should this be quiet instead of silent? Turning off warnings with 'silent' seems like it might make debugging things harder in the future.